.restyled.yaml

```diff
@@ -0,0 +1,5 @@
+- astyle:
+  arguments:
```

Comment:
:wave: Hi there- I hope you don't mind me popping in. I'm the developer of Restyled and I noticed this configuration is causing an error.

What you've got here is a common mistake, and I hope to make it more discoverable...

You've defined an object with the keys `{ astyle: ..., arguments:, ... }`, but that's not valid. What you need is an object of a single key, the restyler name: `{ astyle: { arguments: ... } }`. Which just means indenting the `arguments` a level further:

```yaml
---
- astyle:
    arguments:
      - -A3
      - -T4
```

Hope this helps!

Reply:
Hi, I was writting form my phone at bed, didn't notice that one.. Thanks you !
Cool app and well implemented, keep the good work :+1: